eng/pipelines/templates/steps/wrapped-use-python.yml

```diff
@@ -0,0 +1,60 @@
+parameters:
+  versionSpec: ''
+
+steps:
+  # use python 3.8 for tooling. packaging. platform.
+  - task: UsePythonVersion@0
+    displayName: "Use Python 3.8"
+    inputs:
+      versionSpec: 3.8
+
+  - pwsh: |
+      pip install packaging
+    displayName: Prep Environment
+
+  # select the appropriate version from manifest if present
+  - task: PythonScript@0
+    displayName: 'Check Version Spec Against Known Versions'
+    inputs:
+     scriptPath: 'scripts/devops_tasks/resolve_installer_location.py'
```

Comment:
Why not combine this installer location as well as the powershell script below into one script step? Whether it be PS or Python it seems l like it might be useful for local usage also.

Reply:
It's not. These are the installers specifically intended for the devops or actions agents.

Reply:
That being said, I'll definitely see if that works.